frontend/public/index.html

```diff
@@ -9,7 +9,7 @@
     <!-- Any updates to this meta tag must be reflected in the applications nginx.conf.template for these changes to take effect in openshift-->
     <meta
       http-equiv="Content-Security-Policy"
-      content="default-src 'self'; frame-ancestors 'self'; img-src 'self' data: https://openmaps.gov.bc.ca/ https://*.tile.openstreetmap.org/ https://server.arcgisonline.com/; frame-src 'self' https://*.oidc.gov.bc.ca http://localhost:8080; connect-src 'self' https://*.oidc.gov.bc.ca/ http://localhost:8080/ https://openmaps.gov.bc.ca/ https://server.arcgisonline.com/; form-action 'self'; style-src 'self' 'unsafe-hashes' 'unsafe-inline';"
+      content="default-src 'self'; frame-ancestors 'self'; img-src 'self' data: https://openmaps.gov.bc.ca/ https://*.tile.openstreetmap.org/ https://server.arcgisonline.com/; frame-src 'self' https://*.oidc.gov.bc.ca http://localhost:8080; connect-src 'self' https://*.oidc.gov.bc.ca/ http://localhost:8080/ https://openmaps.gov.bc.ca/ https://server.arcgisonline.com/; form-action 'self'; style-src 'self' 'unsafe-hashes' 'unsafe-inline'; script-src 'self' 'sha256-yt+SNVxRkIi6H6yb7ndFuZM1esMX9esg3UpRHaTsyVk=';"
```

Comment:
fixes an issue where the UI would freeze when saving modifications when developing locally.

Reply:
Can you explain how this works?  For some reason I think I remember SonarQube complaining about a sha256 like this before.

Reply:
all of the scripts used by the application are packaged up by webpack into a single merged file used by the dev application. that file can then be hashed to the above value. Note, this does not seem to include styled components scripts, just static included js, primarily from 3rd party components.

The idea behind including this here is that if something is introduced that breaks these hashes, it'll fail the application in the developer's local environment. Hopefully this will clue them into the fact that they will need to update the actual content-security-policy that is provided by nginx.